draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -7912,7 +7917,14 @@ Accept-Ranges: bytes
 Accept-Ranges: none
 </sourcecode>
 <t>
-   to advise the client not to attempt a range request.
+   to advise the client not to attempt a range request. The range unit "none"
+   is reserved for this purpose.
```

Comment:
You need to do something like this to properly qualify it for the possibility that a proxy added the value:

```suggestion
   to advise the client not to attempt a range request on the current request path. The range unit "none"
   is reserved for this purpose.
```